verify-merge.py

```diff
@@ -10,19 +10,24 @@
 def verify():
     global args, workdir
     os.chdir('gitian-pubkeys')
-    print('Importing pubkeys...')
-    keys = [f for f in glob.glob("*.asc", recursive=True)]
-    for key in keys:
-        subprocess.check_call([GPG, '--import', key])
-    print('Refreshing pubkeys...')
-    subprocess.check_call([GPG, '--refresh'])
+    if args.import_keys:
+        print('Importing pubkeys...')
+        keys = [f for f in glob.glob("*.asc", recursive=True)]
+        for key in keys:
+            subprocess.check_call([GPG, '--import', key])
+    if args.refresh_keys:
+        print('Refreshing pubkeys...')
+        subprocess.check_call([GPG, '--refresh'])
     os.chdir('../../gitian-builder')
-    print('\nVerifying '+args.version+' Linux\n')
-    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-linux', '../monero/contrib/gitian/gitian-linux.yml'])
-    print('\nVerifying '+args.version+' Windows\n')
-    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-win', '../monero/contrib/gitian/gitian-win.yml'])
-    print('\nVerifying '+args.version+' MacOS\n')
-    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-osx', '../monero/contrib/gitian/gitian-osx.yml'])
+    if os.path.isdir(workdir + '/' + args.version + '-linux'):
```

Comment:
Repetition can be reduced here with a simple loop:
```python
    for label, id in [("Linux","linux"), ("Windows","win"), ("MacOS","osx")]:
        if os.path.isdir(workdir + '/' + args.version + '-' + id):
            print('\nVerifying '+args.version+' ' + label + '\n')
            subprocess.check_call(['bin/gverify', '-v', '-d', workdir, '-r', args.version+'-'+id, '../monero/contrib/gitian/gitian-'+id+'.yml'])
```

Reply:
Looks good to me, changed it.

Reply:
I'll push the change later today.

Reply:
Pushed.